osf/migrations/__init__.py

```diff
@@ -126,6 +129,25 @@ def update_permission_groups(sender, verbosity=0, **kwargs):
         update_provider_auth_groups(verbosity)
 
 
+def update_storage_regions(sender, verbosity=0, **kwargs):
+    if getattr(sender, 'label', None) == 'osf':
```

Comment:
Out-of-PR-scope comments: may worth another clean-up PR

It would be great if can define the label and name in settings we can use everywhere including the `AppConfig` [definition](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/apps.py#L9-L10).

And an unrelated question just out of curiosity for both you and @jwalz : do you know why we define the `osf` app config class shadowing the Django `AppConfig`.

```python
...
from django.apps import AppConfig as BaseAppConfig
...
class AppConfig(BaseAppConfig):
    name = 'osf'
    app_label = 'osf'
    ...
```